WebContent/WEB-INF/web.xml

```diff
@@ -421,7 +421,7 @@
     <listener-class>com.serotonin.mango.MangoContextListener</listener-class>
   </listener>
   <session-config>
-    <session-timeout>30</session-timeout>
+    <session-timeout>0</session-timeout>
```

Comment:
Please leave the timaout as it was

Reply:
Yes, it was part of another task, and I also commented on it. Turns off the session timeout.

Reply:
Withdrawal of changes